ams/ams-server/src/main/java/com/netease/arctic/ams/server/optimize/IcebergFullOptimizePlan.java

```diff
@@ -88,9 +88,17 @@ protected boolean partitionNeedPlan(String partitionToPath) {
         TableProperties.SELF_OPTIMIZING_MAJOR_TRIGGER_DUPLICATE_RATIO_DEFAULT);
     // delete files total size reach target_size * duplicate_ratio
     if (deleteFilesTotalSize > targetSize * duplicateRatio) {
-      LOG.debug("{} ==== need native Full optimize plan, partition is {}, " +
+      LOG.debug("{} ==== need native Full optimize plan by {} > {} * {} , partition is {}, " +
               "delete files totalSize is {}, target size is {}",
-          tableId(), partitionToPath, deleteFilesTotalSize, targetSize);
+          tableId(), deleteFilesTotalSize, targetSize, duplicateRatio,
+        partitionToPath, deleteFilesTotalSize, targetSize);
+      return true;
+    }
+
+    if (checkOptimizeInterval(partitionToPath)) {
```

Comment:
I think it is not appropriate to trigger a full compaction when my iceberg table has not been updated.

Reply:
Thank you very much for your comment.

The existing full merge mechanism for iceberg is only controlled based on `self self-optimizing.major.trigger.duplicate-ratio`. in practice,some tables have a small number of delete, which can lead to some small file issues. We can only manually trigger through `self-optimizing.major.trigger.duplicate-ratio=0` full merge.

Add `self-optimizing.full.trigger.interval` support for iceberg full merge. Users can decide based on application scenarios
use `self-optimizing.major.trigger.duplicate-ratio` or `self-optimizing.full.trigger.interval` to trigger a full merge.
I think this is a reasonable demand.

Reply:
Thanks for the explain, I see.